src/app/devices/devices.component.html

```diff
@@ -127,7 +127,7 @@ <h3 *ngIf="isNoData()" class="flex justify-center">
             <button mat-icon-button matTooltip="Restart the device" (click)="$event.stopPropagation(); sendPowerAction(element.guid,5)">
               <mat-icon>restart_alt</mat-icon>
             </button>
-            <button mat-icon-button matTooltip="Deactivate the Device" (click)="$event.stopPropagation(); sendDeactivate(element.guid)">
+            <button mat-icon-button matTooltip="{{selectRemoveDeviceLabel()}}" (click)="$event.stopPropagation(); sendDeactivate(element.guid)">
```

Comment:
I would prefer to use property binding instead of interpolation for this attribute.

Reply:
Good idea I like this better too